test/typescript.test.ts

```diff
@@ -17,14 +17,12 @@ describe('typescript', () => {
       commandsDir: p('src/commands'),
     })
   })
-
   withConfig
   .stdout()
-  .it('runs ts command and prerun hooks', async ctx => {
+  .it('runs ts command and prerun and postrun hooks', async ctx => {
```

Comment:
Might as well have this as a separate test. Also, what about if the command errors? Add a test for that.  I'm assuming the hook won't fire. Is that the right behavior?